app/src/androidTest/java/org/mozilla/focus/helper/RecyclerViewMatcher.java

```diff
@@ -0,0 +1,72 @@
+package org.mozilla.focus.helper;
+
+import android.content.res.Resources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class RecyclerViewMatcher {
```

Comment:
we already have this https://github.com/mozilla-tw/FirefoxLite/blob/58f460ab11502e3598bf751e518494c9ea69dfba/app/src/androidTest/java/org/mozilla/focus/utils/RecyclerViewTestUtils.java#L27